tsconfig.json

```diff
@@ -3,12 +3,15 @@
       "declaration": true,
       "forceConsistentCasingInFileNames": true,
       "importHelpers": true,
-      "module": "commonjs",
+      "module": "es2015",
```

Comment:
this is a breaking change. although webpack 3 can handle es2015 modules fine, users may be using other tools that can't. if you want to make this change, it needs to be additive. blueprint publishes both commonjs and es2015 modules in the same NPM package.